src/Rooms.js

```diff
@@ -36,19 +45,21 @@ class Rooms extends React.Component {
     return (
         <div className="table">
           <h1>List of rooms</h1>
-          <div className="info">{this.state.roomData.map(function(item, i){
-            if(item.roomType === 2){
-              item.topic = "Private Room"
-              item.roomType = "Private"
-            }
-            else if(item.roomType === 3) {
-              item.topic = "Confidential Room"
-              item.roomType = "Confidential"
-            }
-            else {
-              item.roomType = "Public"
-            }
-              return(
+          <div className="info">
+
+            {this.state.roomData.map(function(item, i){
+              if(item.roomType === 2){
+                item.topic = "Private Room"
+                item.roomType = "Private"
+              }
+              else if(item.roomType === 3) {
+                item.topic = "Confidential Room"

```

Comment:
You can do this and sort the array by iterating on the list only once. So remove it from here.